apps/charterafrica/src/pages/index.page.js

```diff
@@ -42,6 +42,19 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
     fallbackLocale: defaultLocale,
   });
 
+  const { docs: homePage } = await payload.findPage("home", "pages", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  const { blocks } = homePage[0] ?? {
+    blocks: [],
+  };
+
+  const ecosystemItems = blocks.filter(
+    (block) => block.blockType === "ecosystem"
+  )[0]?.items;
```

Comment:
Why not just map `blockType` to `slug` and be done with?

```suggestion
  const { blocks: pageBlocks } = homePage[0] ?? {
    blocks: [],
  };
  const blocks = pageBlocks.map(({ blockType, ...other }) => ({...other, slug: blockType }));
  const ecosystem = blocks.find((block) => block.slug === "ecosystem");
```

And then you'll add the whole ecosystem in the return as is. i.e.
```js
  return {
    props: {
      blocks: [{
        ...other blocks
      },
      ecosystem,
      ],
      ...other props
``